specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/azureWebCategory.json

```diff
@@ -0,0 +1,144 @@
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/AzureWebCategories": {
+      "get": {
+        "tags": [
+          "AzureWebCategories"
+        ],
+        "operationId": "AzureWebCategories_ListAll",
+        "description": "Gets all the Azure Web Categorys in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of Azure Web Category resources.",
+            "schema": {
+              "$ref": "#/definitions/AzureWebCategoryListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all Azure Web Categorys for a given subscription": {
+            "$ref": "./examples/AzureWebCategoriesListBySubscription.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureWebCategoryPropertiesFormat": {
+      "properties": {
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the Azure Web Category resource."
+        },
+        "group": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the group that the category belongs to."
+        }
+      },
+      "description": "Azure Web Category Properties."
+    },
+    "AzureWebCategory": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AzureWebCategoryPropertiesFormat",
+          "description": "Properties of the Azure Web Category."
+        },
+        "name": {
+         "type": "string",
+         "description": "Name of the Azure Web Category."
+       },
+       "id": {
+         "type": "string",
+         "readOnly": true,
+         "description": "The ID of the Azure Web Category."
+       },
+       "etag": {
+         "readOnly": true,
+         "type": "string",
+         "description": "A unique read-only string that changes whenever the resource is updated."
+       },
+       "type": {
+         "readOnly": true,
+         "type": "string",
+         "description": "Type of the resource."
+       }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
```

Comment:
the "Resource" definition in network.json has name, id, type, location, and tags. This is not a tracked resource so it shouldn't have location or tags and it shouldn't repeat properties it is inheriting from another definition here in the AzureWebCategory definition

Reply:
I added this because my local validation failed with a message below when running autorest --azure-validator --openapi-type=arm --tag=package-2020-06 ..\..\..\readme.md

ERROR (RequiredPropertiesMissingInResourceModel/R2020/ARMViolation): Model definition 'AzureWebCategory' must have the properties 'name', 'id' and 'type' in its hierarchy and these properties must be marked as readonly.
    - file:///F:/ws/Networking/azure-rest-api-specs/specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/azureWebCategory.json:93:4 ($.definitions.AzureWebCategory)

Reply:
You were getting that error because you haven't marked "name" as readonly.  Please do that and remove the reference to the "Resource" definition (or reference a definition that only has id, name, and type and remove those properties on your definition)